src/app/p-common/menubar/menubar.component.ts

```diff
@@ -47,11 +47,17 @@ export class MenubarComponent implements OnInit {
         visible: !!this.userRole,
       },
       {
-        label: 'Excel',
-        icon: 'pi pi-fw pi-table',
+        label: 'Podsumowanie',
+        icon: 'pi pi-fw pi-list',
         routerLink: [ '/excel' ],
         visible: this.userRole === 'ADMIN',
       },
+      {
+        label: 'Wykresy',
+        icon: 'pi pi-fw pi-chart-bar',
+        // routerLink: [ '/charts' ],
```

Comment:
ejej to moje

Reply:
ojoj no wiem, tylko zamarkowałem na belce

Reply:
no fajnie, ale 1. to nie działa, 2. zmiany w commicie niezwiązane z zadaniem (pomijam formatowanie całego pliku poza tym co piszesz), 3. będę miał konflikty. Albo ty będziesz miał, zależy kto pierwszy